games-util/gamemode/gamemode-1.2.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit ninja-utils meson multilib-minimal
+
+DESCRIPTION="Optimise Linux system performance on demand"
+HOMEPAGE="https://github.com/FeralInteractive/gamemode"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/FeralInteractive/gamemode.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/FeralInteractive/gamemode/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~x86 -*"
+fi
+
+LICENSE="BSD"
+SLOT="0"
+IUSE=""
+
+RDEPEND="
+	>=sys-apps/systemd-236[${MULTILIB_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+src_configure() {
+	multilib-minimal_src_configure
+}
+
+multilib_src_configure() {
+	local myconf=()
+	if ! multilib_is_native_abi; then
+		myconf+=(
+			-Dwith-examples=false
+			-Dwith-daemon=false
+		)
+	fi
+	meson_src_configure "${myconf[@]}"
+}
+
+multilib_src_compile() {
+	eninja
+}
+
+multilib_src_install() {
+	DESTDIR="${D}" eninja install
+	mkdir -p "${D%/}/etc/security/limits.d"
+	cat >"${D%/}"/etc/security/limits.d/45-gamemode.conf <<-EOF
+		@games - nice -10
+	EOF
+}
+
+pkg_postinst() {
+	einfo "GameMode needs a kernel capable of SCHED_ISO to use its soft realtime feature."
```

Comment:
Please use `linux-info` eclass to check for kernel features. Grep some ebuilds to see how to use it.

Reply:
Well, I've still put such a check in `pkg_pretend`.

Reply:
Please make it more explanatory. The kernel check should make it explicit that the support is optional; it would probably be helpful to mention examples of kernels that have it.

Reply:
Okay, I think I figured how to use `CONFIG_CHECK` better, and the warning is now less explicit. I also amended the postinst log to give examples of kernels.